includes/class-wpfaevent-deactivator.php

```diff
@@ -1,36 +1,14 @@
 <?php
-
-/**
- * Fired during plugin deactivation
- *
- * @link       https://fossasia.org
- * @since      1.0.0
- *
- * @package    Wpfaevent
- * @subpackage Wpfaevent/includes
- */
-
```

Comment:
no need to delete all the comments. It actually bloats the PR and makes it close to unreviewable.